CHANGELOG.md

```diff
@@ -6,7 +6,9 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Added
 
-- Nothing.
+- The configuration that is written is now sorted by its keys.
+  This new feature is enabled by default. Sorting can be disabled by 
+  calling `->setSortingEnabled(false);`
```

Comment:
Where would you call this, exactly?

Generally speaking, this module is used in development by zf-apigility-admin. As such, you would need to call it on the instance itself, which means using a custom factory, or a delegator factory. You should provide examples of that both here in the changelog, as well as the README.md file.

Reply:
Also, this should note that this feature is available since 1.4.0.

Reply:
Regarding the version: Isn't it clear that it's part of 1.4.0 since it's under the `1.4.0 - TBD` header?

Regarding documentation, is it really up to to this module to dictate how this method should be used? For all we know someone could install the package via composer and just use ConfigResource class without the service manager stuff. It's no problem for me to add examples but it feels a bit out of scope. I do agree that the current description is unclear. I've changed the last line to "Sorting can be disabled by calling `setSortingEnabled(false);` on the `ConfigResource` class.".

If you feel really strong about adding examples, I'll add them to the README, no problem.